plugins.xml

```diff
@@ -7047,18 +7047,34 @@
 		</release>
 	</plugin>
 	<plugin category="generic" product="reviewerCredits">
-		<name locale="en">ReviewerCredits Plugin</name>
-		<name locale="en_US">ReviewerCredits Plugin</name>
+		<name locale="en">Reviewer Credits Plugin</name>
+		<name locale="en_US">Reviewer Credits Plugin</name>
 		<homepage>https://gitlab.com/reviewercredits/reviewercredits-ojs-plugin</homepage>
-		<summary locale="en">This plugin enable the integration with ReviewerCredits (www.reviewercredits.com).</summary>
-		<summary locale="en_US">This plugin enable the integration with ReviewerCredits (www.reviewercredits.com).</summary>
-		<description locale="en"><![CDATA[This plugin enables the integration with ReviewerCredits using REST APIs. The main goal is to find reviewers and exchange peer review data with the ReviewerCredits platform.]]></description>
-		<description locale="en_US"><![CDATA[This plugin enables the integration with ReviewerCredits using REST APIs. The main goal is to find reviewers and exchange peer review data with the ReviewerCredits platform.]]></description>
+		<summary locale="en">This plugin enable the integration with Reviewer Credits (www.reviewercredits.com).</summary>
+		<summary locale="en_US">This plugin enable the integration with Reviewer Credits (www.reviewercredits.com).</summary>
+		<description locale="en"><![CDATA[This plugin enables the integration with Reviewer Credits using REST APIs. The main goal is to find reviewers and exchange peer review data with the Reviewer Credits platform.]]></description>
+		<description locale="en_US"><![CDATA[This plugin enables the integration with Reviewer Credits using REST APIs. The main goal is to find reviewers and exchange peer review data with the Reviewer Credits platform.]]></description>
 		<maintainer>
 			<name>Emre Danisan</name>
 			<institution>Reviewer Credits GmbH</institution>
 			<email>emre.danisan@reviewercredits.com</email>
 		</maintainer>
+		<release date="2024-12-03" version="2.2.0.1" md5="2874f24c908823ef957528797f41da49">
+			<package>https://www.reviewercredits.com/wp-content/uploads/reviewer-credits/ojsplugin/v2/reviewercredits-ojs-plugin-stable-3_3_0.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>~3.3.0.0</version>
```

Comment:
@ReviewerCredits, this won't appear to be compatible with OJS 3.3.0-0 to 3.3.0-10 as we didn't yet support semantic versioning wildcards in those releases. Can you add those versions manually to the list? (See other plugins for examples.)

Reply:
Hi @asmecher , should be done now. Thanks!